src/schema/sources.ts

```diff
@@ -818,8 +911,29 @@ export const resolvers: IResolvers<any, Context> = {
 
       return getSourceById(ctx, info, sourceId);
     },
+    removeMember: async (
+      _,
+      { sourceId, memberId }: RemoveMemberArgs,
+      ctx,
+    ): Promise<GQLEmptyResponse> => {
+      await ensureSourcePermissions(
+        ctx,
+        sourceId,
+        SourcePermissions.MemberRemove,
+        memberId,
+      );
+
+      await ctx
+        .getRepository(SourceMember)
+        .delete({ sourceId, userId: memberId });
+
+      return { _: true };
+    },
   }),
   Source: {
     permalink: (source: GQLSource): string => getSourceLink(source),
   },
+  SourceMember: {
+    roleRank: (member: SourceMember) => roleRank[member.role],
```

Comment:
Not needed due to graphorm

Reply:
The snapshot is failing, due to the property always returning null.

Reply:
Can you try and remove the transform part of this field in graphorm? Maybe this is the issue

Reply:
I tried it but no luck. Also tried adding the query on the custom relation like below:
```
currentMember: {
        relation: {
          isMany: false,
          customRelation(ctx, parentAlias, childAlias, qb): QueryBuilder {
            return qb
              .select('*')
              .addSelect(this.SourceMember.fields.roleRank.select, '"roleRank"') // added this one
              .from(SourceMember, 'sm')
              .where(`sm."userId" = :userId`, { userId: ctx.userId })
              .andWhere(`sm."sourceId" = "${parentAlias}".id`);
          },
        },
```

Reply:
I'll try to see if I can fix that. Sec please